src/datenversender.c

```diff
@@ -57,6 +57,11 @@ IG_Status init_versender(IG_Datenversender * sender) {
 int rc;
 int err;
 pthread_t tid[1];
+IG_Status rt;
+
+rt = IG_Config_MQTT_get_ClientConfig(sender->config, /*IG_ConfigResponse  */);
```

Comment:
Don't know how to get access to "IG_ConfigResponse * response"... was trying to add the get_ClientConfig inside the init_versender function. 
I need you to explain me the archtiecture of the config_mqtt once again.